src/editor.js

```diff
@@ -1520,11 +1520,12 @@ export class OdooEditor extends EventTarget {
 
             if (closestTag === 'LI') {
                 this._applyCommand('indentList', ev.shiftKey ? 'outdent' : 'indent');
-                ev.preventDefault();
             } else if (closestTag === 'TABLE') {
                 this._onTabulationInTable(ev);
-                ev.preventDefault();
+            } else if (!ev.shiftKey) {
+                this.execCommand('insertText', '\u00A0 \u00A0\u00A0');
             }
+            ev.preventDefault();
```

Comment:
If none of the if / else if matches, I don't think we want to `preventDefault()`.
In this case we have to keep the call inside the conditions.

Reply:
The task I'm handling here is called "Tab should not move focus outside of the editor". The default behavior of TAB is (except for tables and lists, which we handle separately) to move the focus outside of the editor. So I preventDefault every time so it never happens.

Reply:
ok then, my bad.